tensorrt_llm/_torch/visual_gen/models/wan/pipeline_wan.py

```
@@ -459,12 +496,35 @@ def forward_fn(
+
             return current_model(
                 hidden_states=latents,
                 timestep=timestep,
                 encoder_hidden_states=encoder_hidden_states,
             )
 
+        # Pin reference image to latent after each scheduler step (Wan 2.2 5B I2V only)
+        post_step_fn = None
+        if self.is_wan22_5b and is_i2v:
+
+            def post_step_fn(x):
```

Flake8 (7.3.0)

[error] redefinition of unused 'post_step_fn' from line 520

(F811)